Header header
float32 x
float32 y
float32 height
float32 width
uint8 category_id
string category_name